<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TmsPassword</class>
 <widget class="QDialog" name="TmsPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>748</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕코딩</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>50</y>
     <width>54</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>50</y>
     <width>54</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>50</y>
     <width>54</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>54</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>50</y>
     <width>54</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>318</x>
     <y>50</y>
     <width>54</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>50</y>
     <width>54</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>436</x>
     <y>50</y>
     <width>54</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>50</y>
     <width>54</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>554</x>
     <y>50</y>
     <width>54</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>비밀번호</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>144</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>144</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>144</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>144</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>144</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>144</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>440</y>
     <width>144</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>144</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>144</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>144</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>144</width>
     <height>293</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="new_tmsview.qrc">
     <normaloff>:/images/backspace.png</normaloff>:/images/backspace.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_enter">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>590</y>
     <width>443</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="new_tmsview.qrc">
     <normaloff>:/images/enter.png</normaloff>:/images/enter.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>144</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="new_tmsview.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="new_tmsview.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>TmsPassword</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_enter</sender>
   <signal>clicked()</signal>
   <receiver>TmsPassword</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
